### PR TITLE
[Changelog] Update the changelog for the 0.14.0-alpha.2 pre-release.

### PR DESCRIPTION
**Pull request by q82419**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: .CurrentChangelog.md [hunk 1]
```diff
@@ -68,7 +72,7 @@ Features:
 * [Tools]: Print the plug-in versions when using the `--version` option.
 * [Installer]: Enabled `ggml-blas` and `rustls` plugin supporting (#3032) (#3108).
 * [WASI-NN] ggml backend:
-  * Bump llama.cpp to b1743
+  * Bump llama.cpp to b2037.
```
Comment (hydai) [inline .CurrentChangelog.md:75]: 2230 now.

File: .CurrentChangelog.md [hunk 2]
```diff
@@ -77,14 +81,21 @@ Features:
   * Default enable Metal on macOS.
   * Introduce `load_by_name_with_config()` to load model with metadata.
   * Introduce single token inference by `compute_single`, `get_output_single`, and `fini_single`
-  * Add some llama errors to WASI-NN
+  * Add some llama errors to WASI-NN.
     * `EndOfSequence`: returned when encounter `<EOS>` token on single token inferece.
     * `ContextFull`: returned when the context is full.
     * `PromptTooLong`: returned when the input size is too large.
+  * Support Llava inference.
```
Comment (hydai) [inline .CurrentChangelog.md:88]: One more model, `Gemma`, is supported after shipping Llava.

**Review by hydai — CHANGES_REQUESTED** [review]

File: .CurrentChangelog.md [hunk 3]
```diff
@@ -77,14 +81,21 @@ Features:
   * Default enable Metal on macOS.
   * Introduce `load_by_name_with_config()` to load model with metadata.
   * Introduce single token inference by `compute_single`, `get_output_single`, and `fini_single`
-  * Add some llama errors to WASI-NN
+  * Add some llama errors to WASI-NN.
     * `EndOfSequence`: returned when encounter `<EOS>` token on single token inferece.
     * `ContextFull`: returned when the context is full.
     * `PromptTooLong`: returned when the input size is too large.
+  * Support Llava and Gemma inference.
```
Comment (hydai) [inline .CurrentChangelog.md:88]: Gemma should be a standalone line. It will not use the following two options.

**Review by hydai — CHANGES_REQUESTED** [review]

**Review by hydai — APPROVED** [review]

**Pull request merged**